mm2src/lp_ordermatch.rs

```diff
@@ -3091,6 +3093,7 @@ struct MakerOrderUpdateReq {
     base_nota: Option<bool>,
     rel_confs: Option<u64>,
     rel_nota: Option<bool>,
+    save_in_history: Option<bool>,
```

Comment:
It might be a matter of taste, but I think that it can be confusing a bit that users and developers will need to keep in mind that if they created an order with `save_in_history:false` they also need to set this flag in all update requests. They will have to record this state somehow as it won't be returned in order_status which adds a bit more complexity.

Maybe it will be easier to add this field directly to the order structs with `#[serde(default=true)]`?

Reply:
yeah, this will be more straightforward. should the `save_in_history` param be kept in `update_maker_order` request in case the user wanted to save the order in the update request after setting it to false in `set_price` ? my opinion is it's too much and it should be set in `set_price` only.

Reply:
> my opinion is that this is too much and it should be set in set_price only.

I agree, it's fine to set it once in setprice/buy/sell :slightly_smiling_face: